yktracec/src/jitmodbuilder.cc

```diff
@@ -1052,9 +1052,7 @@ class JITModBuilder {
   Module *createModule() {
     size_t CurBBIdx;
     size_t CurInstrIdx;
-    // Iterate over the blocks of the trace. The first block contains the
```

Comment:
Shall we add a comment saying what the first block should be now?

You might be able to add an assertion that the first block is a successor of the block containing the control point?

Reply:
TBH, I don't know what the first block should be, nor do I know how to encode that assertion!

Reply:
> I don't know what the first block should be

A successor of the block containing the control point call.

>  nor do I know how to encode that assertion

OK, not worth worrying about.

Reply:
Is it always a direct successor. I guess I assumed it could be a function call? Anyway, maybe not a big worry?

Reply:
You're right, it could be an unmappable block inside a function call.